### ISSUE TITLE
[4.x]: Error uploading a jpg file to via Asset field or directly to volume

### ISSUE THREAD

**Issue by benjaminkohl** [craftcms/cms#10806]
### Description

I am attempting to upload a jpg file via an Asset field and also via the Assets page in the control panel and I see the following error response to the following request in my browser's developer console:

```
POST http://c4b.local/index.php?p=admin%2Factions%2Fassets%2Fupload

Response:
{"message":"Cannot assign Imagine\\Gd\\Imagine to property craft\\image\\Raster::$_instance of type ?Imagine\\Imagick\\Imagine"}

```

### Steps to reproduce

1. Attempt to upload a jpg image via an Asset field.
OR
1. Attempt to upload a jpg image directly to a volume via the Assets page in the control panel.

### Actual behavior

The upload progress bar goes to 100% but doesn't disappear and doesn't report that an error occurred. It just remains on the screen until I refresh the page.

### Craft CMS version

Craft Pro 4.0.0-beta.1

### PHP version

8.0.8

### Operating system and version

Darwin 20.6.0

### Database type and version

MySQL 5.7.34

### Image driver and version

GD 8.0.8

### Installed plugins and versions

None.

**Comment by angrybrad** [comment]
This was resolved in [4.0.0-beta.2](https://github.com/craftcms/cms/blob/4.0/CHANGELOG.md#fixed-1), but you might as well update to 4.0.0-beta.3.

**Comment by benjaminkohl** [comment]
Doh! I didn't even realize there was a beta.2 or beta.3. Thanks!
